Remora.Discord.Interactivity/Constants.cs

```diff
@@ -34,4 +34,9 @@ public static class Constants
     /// Gets the name of the interaction command tree.
     /// </summary>
     public const string InteractionTree = "remora::interactivity";
+
+    /// <summary>
+    /// Gets the prefix used for passing the state.
+    /// </summary>
+    public const string StatePrefix = "$!";
```

Comment:
Let's swap this over to `0x1D` (ASCII Group Separator), since it's highly unlikely to appear in plaintext state information.

Reply:
Actually, nevermind. It's best done later in a rework.